src/screens/manage-accounts/manage-hd-accounts/manage-hd-accounts.tsx

```diff
@@ -21,7 +26,11 @@ export const ManageHdAccounts = () => {
   const styles = useManageHdAccountsStyles();
   const revealSelectBottomSheetController = useBottomSheetController();
 
-  const hdAccounts = useHdAccountsListSelector();
+  const hdAccounts = useHdAccountListSelector();
+  const { setSearchValue, filteredAccountList } = useFilteredAccountList(hdAccounts);
+
+  const debouncedSetSearch = debounce(setSearchValue);
```

Comment:
the same behaviour in 2 places.  
move `debouncedSetSearch` inside `useFilteredAccountList`